model/registry/event.yaml

```
@@ -16,3 +16,18 @@ groups:
           separation of semantics for events in separate domains like browser, mobile, and
           kubernetes.
         examples: ['browser.mouse.click', 'device.app.lifecycle']
+      - id: summary
+        type: string
+        stability: experimental
+        brief: >
+          Human-readable summary representing an event.
+        note: >
+          Events are identified by an `event.name` and a set of attributes and fields in the body that carry specific
+          meaning. However, since these events will be combined with other logs, `event.summary` allows a
+          centralized logging system to display a human-readable representation of the event.
+
+          When summaries are generated, they are not expected to include every attribute and field that is part of the
+          event but could contain those that are meaningful for a human operator when visually navigating a centralized
+          log system. Instrumentation libraries that produce events defined within the standard OpenTelemetry are not
+          expected to add an `event.summary` attribute, as these are well-known events.
```

Could we make it normative? E.g.
```suggestion
          It's NOT RECOMMENDED to record full string representation of the event on the `event.summary` attribute. The summary SHOULD be concise and human-readable as it's expected to be used by a human operator when visually navigating a centralized log system.
```

I removed 

> Instrumentation libraries that produce events defined within the standard 
OpenTelemetry are not expected to add an `event.summary` attribute, as these are well-known events.

I'm not sure how it could work - if we have 100+ events defined, no backend would be able to special-case each event and define human-readable representation of it. I.e. I don't understand why we would allow to define a summary for at least some standards events.